common/types/pb/file_test.go

```diff
@@ -1,3 +1,17 @@
+// Copyright 2018 Google LLC
```

Comment:
2022?

Reply:
This file was created in 2018, but didn't have the copyright. Should I just do 2022 anyway?

Reply:
Updated.